bika/lims/browser/dashboard/dashboard.py

```diff
@@ -438,9 +439,21 @@ def get_analysisrequests_section(self):
         query['review_state'] = ['published', ]
         out.append(self._getStatistics(name, desc, purl, catalog, query, total))
 
+        # Analysis Requests to be printed
+        if self.context.bika_setup.getPrintingWorkflowEnabled():
+            name = _('Analysis Requests to be printed')
+            desc = _("To be printed")
+            purl = 'analysisrequests?analysisrequests_getPrinted=0'
+            query['getPrinted'] = '0'
+            if 'review_state' in query:
+                del query['review_state']
```

Comment:
I'd add the `review_state` in which the printed option makes sense (`published`) instead of removing it:
```python
query['getPrinted'] = 0
query['review_state'] = ['published',]
```